docs/source/architecture/rest_api.rst

```diff
@@ -2,313 +2,47 @@
-provided by specific :doc:`../rest_api/error_codes` in the JSON response
-envelope itself (see below).
-
-.. list-table::
-   :widths: 4, 16, 60
-   :header-rows: 1
-
-   * - Code
-     - Title
-     - Description
-   * - 200
-     - OK
-     - The requested resources were successfully fetched. They are included in
-       the ``"data"`` property of the response envelope.
-   * - 201
-     - Created
-     - The POSTed resources were submitted and created/committed successfully. A
-       ``"link"`` is included to the resources.
-   * - 202
-     - Accepted
-     - The POSTed resources were submitted to the validator, but are not yet
-       committed. A ``"link"`` to check the *status* of the submitted resources
-       is included. If told to wait for commit, but timed out, the status at the
-       moment of timing out is also included in the ``"data"`` property.
-   * - 400
-     - Bad Request
-     - Something in the client's request was malformed, and the request could
-       not be completed.
-   * - 404
-     - Not Found
-     - The request was well-formed, but there is no resource with the identifier
-       specified.
-   * - 500
-     - Internal Server Error
-     - Something is broken in the REST API or the validator. If consistently
-       reproducible, a bug report should be submitted.
-   * - 503
-     - Service Unavailable
-     - Indicates the REST API is unable to contact the validator.
-
-
-Data Envelope
-=============
-
 The REST API uses a JSON envelope to send metadata back to clients in a way that
-is simple to parse and easily customized. All successful requests will return
-data in an envelope with at least one of four possible properties:
-
-   * **data** - contains the actual resource or resources being fetched
-   * **head** - id of the head block of the chain the resource was fetched
-     from, this is particularly useful to know if an explicit *head* was not set
-     in the original request
-   * **link** - a link to the resources fetched with both *head* and
-     paging parameters explicitly set, will always return the same resources
-   * **paging** - information about how the resources were paginated, and how
-     further pages can be fetched (see below)
-
-*Example response envelope:*
-
-.. code-block:: json
-
-   {
-     "data": [{"fetched": "resources"}],
-     "head": "65cd...47dd",
-     "link": "http://rest.api.domain/state?head=65cd...47dd"
-   }
+is simple to parse and easily customized. All successful requests return data
+in an envelope that may include the requested resource, chain head ID, a link
+to the requested resource, and paging information (described below).
 
+The JSON response envelope also includes the details needed to parse errors. For
+more information, see the :doc:`../rest_api/error_codes`.
 
 Pagination
-----------
-All endpoints that return lists of resources will automatically paginate
-results, limited by default to 1000 resources. In order to specify what range of
```

Comment:
This should still 100

Reply:
I made the change to 100. Your comment is on the deleted content. Please see the "View" version of rest_api.rst. https://github.com/chenette/sawtooth-core/blob/0f0bb5a81d540810ce38031c7cd1ddf02f332a7b/docs/source/architecture/rest_api.rst

Reply:
I'm bad, this is deleted code.